qemu/tests/balloon_thp.py

```diff
@@ -0,0 +1,44 @@
+import time
+
+from virttest.staging import utils_memory
+from virttest.utils_misc import normalize_data_size
+
+
+def run(test, params, env):
+    """
+    Memory balloon with thp
+
+    1. Boot up a guest with balloon support, record memory fragement
+    2. Make fragement in guest with tmpfs
+    3. check the memory fragement with proc system, should increase
+    4. Do memory balloon the memory size ballooned should be a legal value
+    5. Check the memory fragement with proc system
+    6. Try to make the memory compact
+    7. Check the memory status in guest
+    8. Check the fragement in system should decrease
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    buddy_info_bf = utils_memory.get_buddy_info('0', session=session)['0']
+    session.cmd_output_safe(params['cmd_make_fragement'], timeout=600)
```

Comment:
Pls add some logs to indicate the steps, thanks.

Reply:
I agree, Need some logs to describe step.